library/attachment-viewer/src/main/java/im/vector/lib/attachmentviewer/AttachmentsAdapter.kt

```diff
@@ -42,21 +42,21 @@ class AttachmentsAdapter : RecyclerView.Adapter<BaseViewHolder>() {
         val inflater = LayoutInflater.from(parent.context)
         val itemView = inflater.inflate(viewType, parent, false)
         return when (viewType) {
-            R.layout.item_image_attachment          -> ZoomableImageViewHolder(itemView)
+            R.layout.item_image_attachment -> ZoomableImageViewHolder(itemView)
             R.layout.item_animated_image_attachment -> AnimatedImageViewHolder(itemView)
-            R.layout.item_video_attachment          -> VideoViewHolder(itemView)
-            else                                    -> UnsupportedViewHolder(itemView)
+            R.layout.item_video_attachment -> VideoViewHolder(itemView)
+            else -> UnsupportedViewHolder(itemView)
```

Comment:
You will not convince me that the new format is easier to read. (but I know the advantages).
Applicable to nearly all the `when` statements.
Probably because to me code readability is top priority.

Reply:
It is mine too, but personally I find this easier to read than if they're all aligned. It also may well be the case that this is easier or harder to read based on what you're used to and falls into the realm of subjective, hence, the earlier poll